### PR TITLE
fix bug Call to undefined method Whitecube\NovaPage\Pages\Query::exists()

### PR DESCRIPTION
**Pull request by AbdullahGhanem**

### CODE REVIEW

File: src/Pages/Query.php [hunk 1]
```diff
@@ -108,6 +108,18 @@ public function get($throwOnMissing = false)
...
+    
+    /**
+     * Mimic eloquent's Builder and execute the query
+     *
+     * @param bool $throwOnMissing
+     * @return Illuminate\Support\Collection
+     */
+    public function exists($throwOnMissing = false)
+    {
+        return true;
```
Comment (wshurafa) [inline src/Pages/Query.php:121]: This returns 'true' all the time despite this instance exists or not. It should query the database and return a proper boolean value according to the existence of the model instance.

Comment (tjmckenzie) [reply src/Pages/Query.php:121]: For this check, could we utilize the already mimic'd `get()`?
```suggestion
        $results = $this->get($throwOnMissing);

        return $results->count() === 1;
```

Comment (wshurafa) [reply src/Pages/Query.php:121]: @AbdullahGhanem can you check this comment and apply it?

**Comment by wshurafa** [comment]
This is a fix for issue #93

**Review by toonvandenbos — APPROVED** [review]
Thank you!

**Pull request merged**